docs(cpt): add pricing plans info (#4029)
scaleway/docs-content: faq/cockpit.mdx

**Total:** €6.650 per month with volume discounts instead of €7.800 without volume discounts.

## Why are Cockpit pricing plans being deprecated?

From January 1st 2025, Cockpit is transitioning away from fixed pricing plans to offer you more flexibility and granularity for managing data [retention](/observability/cockpit/concepts/#retention).

> This change allows you to tailor retention settings to your specific needs. This evolution aims at enhancing your experience by providing more customizable options without immediate cost impact, as changing data retention period will be free during Beta.

Reviewer: ```suggestion
This change allows you to tailor retention settings to your specific needs. We hope to enhance your experience by providing more customizable options without immediate cost impact, as changing data retention period will be free during Beta.
```

Reviewer: If there are no fixed pricing plans any more, does it mean it will be free whatever retention period you choose? It's not totally clear to me

Reviewer 2: Yes, longer than default retention will not be billed during the beta phase. Long retention will be billed after that first step, but pricing structure and timeline are still unknown. 
happy to have your inputs to improve the wording and make it clear for everyone !
